common/pulp/common/error_codes.py

```diff
@@ -100,6 +100,9 @@
 PLP0041 = Error("PLP0041", _("Database 'replica_set' config must be specified when more than one "
                              "seed is provided. Refer to /etc/pulp/server.conf for proper use."),
                 [])
+PLP0043 = Error("PLP0043", _("Database 'write_concern' config can only be 'majority' or 'all'. "
+                             "Refer to /etc/pulp/server.conf for proper use."),
+                [])
```

Comment:
Looks like you copy-pasted the one above. This one doesn't need this final argument on a separate line.